src/webviews/webview-side/data-explorer/mainPanel.tsx

```diff
@@ -98,7 +98,7 @@ export class MainPanel extends React.Component<IMainPanelProps, IMainPanelState>
                 fetchedRowCount: -1,
                 filters: {},
                 indexColumn: data.primaryKeys[0],
-                styleReady: false,
```

Comment:
This fixes the rendering not happening. I'll enter another issue to remove the CSS messages entirely.